pkg/reconciliation/reconcile_racks.go

```diff
@@ -2243,6 +2246,52 @@ func (rc *ReconciliationContext) fixMissingPVC() (bool, error) {
 	return false, nil
 }
 
+func (rc *ReconciliationContext) SetFullQueryLogging() result.ReconcileResult {
+	datacenter := rc.GetDatacenter()
```

Comment:
This does not shorten much. Why not just .. dc?

Reply:
Can do.